specification/tags.json

```diff
@@ -106,6 +106,11 @@
     "externalDocs": {
     }
   },
+  {
+    "name": "Payments",
+    "externalDocs": {
+    }
```

Comment:
Lets hide this tag by adding it in https://github.com/MyParcelCOM/api-specification/blob/develop/src/myparcelcom.js#L97